src/pymovements/utils/parsing.py

```diff
@@ -93,6 +92,11 @@
     r'END\s+(?P<timestamp>(\d+[.]?\d*))\s+\s+(?P<types>.*)\s+RES\s+'
     r'(?P<xres>[\d\.]*)\s+(?P<yres>[\d\.]*)\s*',
 )
+TRACKED_EYE_REGEX = re.compile(
```

Comment:
It actually looks like the overall tracking configuration can change, not just the tracked eye.

So instead of just reparsing `tracked_eye`, also reparse the remaining fields (`tracking_mode`, `sampling_rate`, `file_sample_filter`, `link_sample_filter`)

Then you should probably rename the regex to something like `TRACKING_CONFIG_REGEX`.